src/pages/index.astro

```diff
@@ -378,21 +266,118 @@ nf-core list
-                    See a complete list &raquo;
-                </a>
-            </p>
-            <ContributorCarousel client:visible />
-        </div>
-    </div>
+    <ContributorCarousel client:visible />
 </BaseLayout>
+
+<style lang="scss">
+    @import 'bootstrap/scss/bootstrap.scss';
+    @import 'bootstrap-print-css/css/bootstrap-print.min';
+    @import '../styles/_variables.scss';
+    .homepage-header .homepage-header-contents {
+        margin: 3rem auto 0;
+        padding: 0;
+        text-align: center;
+    }
+    .homepage-header-fa-background {
+        position: absolute;
+        top: 10px;
+        left: 30px;
+        font-size: 16em;
+        opacity: 0.2;
+    }
+
+    /* Fancy call to action button on homepage
+Based on https://codepen.io/wintr/pen/beBJBb */
+    .homepage-cta-flex {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin-top: 2rem;
+    }
+    .homepage-cta {
+        width: 100px;
+        height: 100px;
+        line-height: 20px;
+        padding: 30px 0;
+        text-align: center;
+        text-transform: uppercase;
+        letter-spacing: 1px;
+        position: relative;
+        transition: 0.3s all ease;
+        font-size: 14px;
+        color: $nf-core-color;
+        font-weight: bold;
+        z-index: 0;
+    }
+    .homepage-cta:after {
+        transition: 0.3s all ease;
+        content: '';
+        position: absolute;
+        left: 0;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        border-radius: 3px;
+        border: 2px solid $nf-core-color;
+    }
+    .homepage-cta:before {
+        content: '';
+        position: absolute;
+        border-radius: 3px;
+        left: 0;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        -webkit-transform: rotate(45deg);
+        transform: rotate(45deg);
+        background: $gray-200;
+        z-index: -1;
+        transition: 0.3s all ease;
+    }
+    .homepage-cta:hover,
+    .homepage-cta:focus {
+        color: $nf-core-color;
+        text-decoration: none;
+    }
+    .homepage-cta:hover:after,
+    .homepage-cta:focus:after {
+        -webkit-transform: rotate(45deg);
+        transform: rotate(45deg);
+    }
+    .homepage-cta:active {
+        -webkit-transform: scale(1);
+        transform: scale(1);
+        color: $white;
+    }
+    .homepage-cta:active:before {
+        background-color: $nf-core-color;
+    }
+    .homepage-cta:active:after {
+        border-color: $nf-core-color;
+    }
+
+    .searchbar_form {
+        justify-content: center;
+    }
+    .searchbar_form input {
+        border-top-right-radius: 0;
+        border-bottom-right-radius: 0;
+        border-right: none;
+        width: auto;
+    }
+    .searchbar_form button {
+        border-top-left-radius: 0;
+        border-bottom-left-radius: 0;
+    }
+    .homepage_search.searchbar_form {
+        margin-bottom: 3rem;
```

Comment:
can't this rule also be in `.searchbar_form`?

Reply:
Looks like `.searchbar_form` is also used in `search.php`

I guess we won't have such a file in the new site, so yeah I'll just merge it here 👍🏻